### PR TITLE
MD: add ks anaconda useexisting flag when set in aii

### PR DESCRIPTION
**Pull request by kwaegema**
fixes https://github.com/quattor/aii/issues/102

### CODE REVIEW

File: src/test/resources/filesystems.pan [hunk 1]
```diff
@@ -47,5 +47,11 @@ unique template filesystems;
     fs["preserve"] = false;
     append(fs);
 
+    # append md test with useexisting
+    fs=value("/system/filesystems/0");
+    fs["block_device"]=format("md/%s", escape("md/myname"));    
+    fs["useexisting_md"] = true;
```
Comment (stdweird) [inline src/test/resources/filesystems.pan]: where is the PR that allows this in the schema?

File: src/main/perl/MD.pm [hunk 2]
```diff
@@ -284,6 +285,11 @@ sub print_ks
 
     return unless $self->should_print_ks;
 
+    my $exist = '';
```
Comment (stdweird) [inline src/main/perl/MD.pm]: rename to `usexist`, too close to the `exists` function

File: src/test/resources/filesystems.pan (hunk 1 above)
Comment (kwaegema) [inline src/test/resources/filesystems.pan]: @stdweird This is set only by aii-ks for now.. I don't know another way to pass this from there to ncm-lib-blockdevices.

**Comment by stdweird** [comment]
LGTM

**Pull request merged**